api/test/dead_link_filter_test.py

```diff
@@ -27,6 +27,11 @@ def get_redis_connection(*args, **kwargs):
     fake_redis.client().close()
 
 
+@pytest.fixture(autouse=True)
+def turn_off_db_read(monkeypatch):
```

Comment:
Why is this needed for these tests? Can you add a comment explaining it?

Reply:
Added comment in c3681fe.